apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -287,6 +287,28 @@ export default class Utilities {
     });
   }
 
+  public static executeShellCommand(
```

Comment:
Because we need a version that does not need parsing arguments. 

---
In reply to: [116887866](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r116887866) [](ancestors = 116887866,116886431)